lntest/itest/lnd_test.go

```diff
@@ -4356,6 +4357,187 @@ func testSphinxReplayPersistence(net *lntest.NetworkHarness, t *harnessTest) {
 	cleanupForceClose(t, net, carol, chanPoint)
 }
 
+func assertChannelConstraintsEqual(
+	t *harnessTest, want, got *lnrpc.ChannelConstraints) {
+
+	if want.CsvDelay != got.CsvDelay {
```

Comment:
`require.Equal(t, want.CsvDelay, got.CsvDelay)`?

Reply:
Since last PR, I did try to check whether it can be replaced with `require`. However it seems that in the `itest`, `testingT` is wrapped in `harnessTest`, a customized `Fatalf` is used to raise errors, and [extra work is done before calling `testingT.Fatalf`](https://github.com/lightningnetwork/lnd/blob/master/lntest/itest/lnd_test.go#L101).

Using `require.Equal(t.t, want.CsvDelay, got.CsvDelay)` did work, but it bypassed saving profile pages?

Reply:
> Using require.Equal(t.t, want.CsvDelay, got.CsvDelay) did work

My bad, yes passing `t.t` is actually required to work properly

> but it bypassed saving profile pages?

Hmm that is odd, but it is non-blocking. Perhaps we can look into that later.

Reply:
Sure thing. I prefer `require` too so I did a little check.

In the current integration test, the line goes like this,

`harnessTest.Fatalf => SaveProfilesPages => harnessTest.t.Fatalf => c.FailNow()` and exit.

If we use `require`,  it's

`require.Equal(t.t, ...) => t.FailNow()` and exit. So the whole `SaveProfilesPages` is missing.

One way to solve it is that, if we could makes `harnessTest` implementing `TestingT`, and wrap the extra work in `FailNow`, then it would work. Just a thought :)

Reply:
It's bypassed because `harnessTest.Fatalf` is no longer invoked. I'd say let's keep using the harness for now until we can figure out how to integrate things with `require`.